### PR TITLE
Fix datetime cyclic value change

### LINKED ISSUE #98: Date picker hangs because a value cyclic assignment

### PR DESCRIPTION
**Pull request by caleeli**
Resolves #98

### CODE REVIEW

File: src/components/FormDatePicker.vue [hunk 1]
```diff
@@ -71,25 +74,31 @@
...
     },
     methods: {
-      updateFormat() {
-        if (this.dataFormat === 'datetime') {
-          this.config.format = 'MM/DD/YYYY h:mm A';
-        } else  {
-          this.config.format = 'MM/DD/YYYY';
-        }
+      getUserDateFormat() {
+        return ProcessMaker.user.datetime_format.replace(/ |H|:|m|s|z|Z/g, '');
```
Comment (marcoAntonioNina) [inline src/components/FormDatePicker.vue]: check if there is a variable Processmaker

File: src/components/FormDatePicker.vue [hunk 2]
```diff
@@ -71,25 +74,31 @@
...
+      },
+      getUserDateTimeFormat() {
+        return ProcessMaker.user.datetime_format;
```
Comment (marcoAntonioNina) [inline src/components/FormDatePicker.vue]: check if there is a variable Processmaker

**Comment by sanjacornelius** [comment]
This opens up a few bugs with the DateTime picker:

1. The front-end formatting of the DateTime within the input fields is displaying as `2019-09-27T12:00:00-0700` it should display as `09/27/2019 12:00 PM`. This is happening both in the screen-builder & modeler.

2. In the screen builder, the time should default to the current time, currently, it's defaulting to 12:00 PM. The time is displaying correctly in Modeler.

Screen Shot 2019-09-27 at 9 20 09 AM
Screen Shot 2019-09-27 at 9 20 35 AM

File: src/components/FormDatePicker.vue (hunk 1 above)
Comment (caleeli) [reply src/components/FormDatePicker.vue]: fixed

**Comment by caleeli** [comment]
@sanjacornelius about the modeler, it is because the modeler as stand alone does not have a user date time format defined (ProcessMaker.user.datetime_format) and I left the default as the iso 8601 format. But when it is tested in ProcessMaker it will be formatted using the defined in the User Profile 

image

**Pull request merged**